Client/Frontend/Toolbar+URLBar/ToolbarButton.swift

```diff
@@ -10,7 +10,7 @@ class ToolbarButton: UIButton {
 
     var selectedTintColor: UIColor!
     var unselectedTintColor: UIColor!
-    var disabledTintColor = UIColor.Photon.Grey50
+    var disabledTintColor = UIColor.darkGray
```

Comment:
This isn't using the new theming system 🤔 This is a fixed color that we'll deprecate (any colors directly used with `UIColor.`